scripts/refresh_box_tokens.py

```diff
@@ -30,14 +32,18 @@ def main(delta, dry_run):
             )
         )
         if not dry_run:
-            record.refresh_access_token(force=True)
+            try:
+                record.refresh_access_token(force=True)
+            except BoxAuthenticationException as ex:
+                print(ex.message)
```

Comment:
We should use a logger here, and use `scripts.utils.add_file_logger` so that these failures get logged to the migration logs directory.